tests/lax_numpy_indexing_test.py

```diff
@@ -854,16 +854,43 @@ def testJVPOfGradOfIndexing(self):
     self.assertAllClose(expected, primals)
     self.assertAllClose(np.zeros_like(x), tangents)
 
+  def testSimpleIndexingUsesSlice(self):
+    jaxpr = jax.make_jaxpr(lambda x: x[:2, :2])(jnp.ones((3, 4)))
```

Comment:
Instead of just checking the jaxpr, please check the output too

See this function: https://github.com/google/jax/blob/main/tests/array_test.py#L481-L484